frontend/src/v5/ui/controls/assigneesSelect/assigneesSelect.component.tsx

```diff
@@ -89,26 +91,28 @@ export const AssigneesSelect = ({
 		</AssigneesListContainer>
 	);
 	return (
-		<SearchContextComponent fieldsToFilter={['_id', 'firstName', 'lastName', 'job', 'notFoundName']} items={allJobsAndUsersToDisplay}>
-			<AssigneesListContainer onClick={handleOpen} className={className}>
-				<AssigneesSelectMenu
-					open={open}
-					value={value}
-					onClose={handleClose}
-					onOpen={handleOpen}
-					disabled={disabled}
-					multiple={multiple}
-					isInvalid={(v) => invalidValues.includes(v)}
-					onChange={handleChange}
-					{...props}
-				/>
-				<AssigneesValuesDisplay
-					value={valueAsArray}
-					maxItems={maxItems}
-					showEmptyText={showEmptyText}
-					disabled={disabled || !showAddButton}
-				/>
-			</AssigneesListContainer>
-		</SearchContextComponent>
+		<FormControl required={false} disabled={disabled} error={props.error} className={className}>
```

Comment:
This adds a margin to the assignees cell in tabular view
![image](https://github.com/user-attachments/assets/f479f80e-8925-4896-bc73-e4bbf2705a3e)